### ISSUE TITLE
[bug] Changes in JSON serialization not backwards compatible (multi-index)

### ISSUE THREAD

**Issue by dlohmeier** [e2nIEE/pandapower#1908]
### Issue Report Checklist

* [X] Searched the [issues page](https://github.com/e2nIEE/pandapower/issues) for similar reports
* [X] Read the relevant sections of the [documentation](https://pandapower.readthedocs.io/en/latest/about.html) 
* [X] Browse the [tutorials](https://github.com/e2nIEE/pandapower/tree/develop/tutorials) and [tests](https://github.com/e2nIEE/pandapower/tree/develop/pandapower/test) for usefull code snippets and examples of use
* [X] Reproduced the issue after updating with `pip install --upgrade pandapower` (or `git pull`)
* [X] Tried basic troubleshooting (if a bug/error)
    * [X] Restarted interpreter
    * [X] Checked the pythonpath
* [X] Completed the **Problem Description**, **What steps reproduce the problem** and **Versions** sections below

## Problem Description
In the current develop version, there are changes in the JSON encoder that are not backwards compatible (it seems to me that they are related to the possibility to save multi-index dataframes, @SteffenMeinecke). This means that exchanging data between people/servers working on different versions may cause problems.

### What steps reproduce the problem?

1. Make sure pandapower is on develop
2. Load and save any network to JSON file (e.g. one of the example networks, I tried Kerber Dorfnetz)
3. Install pandapower 2.11 (master)
4. Load the saved JSON file

### What is the expected output? What do you see instead?
Nice to have: the same network as was saved. Instead, all normal entries (such as "bus") are now returned as dictionaries with some metadata, not even values that could be used to retrieve the desired output.
Should do: Mention this backwards compatibility break in the changelog.

## Versions

* pandas version: 1.5.3
* networkx version: 3.0
* scipy version: 1.10.1
* numpy version: 1.23.5
* packaging version: 23.0
* tqdm version: 4.65.0
* deepdiff version: 6.2.3
* Operating System name/version: ubuntu 22.04

## Labels (if access available)
* [X] Choose a `labels` at the right side (e.g. `bug`, `controller`, `OPF`...)

**Comment by SteffenMeinecke** [comment]
@dlohmeier you are right. In the time when v2.11.0 und v2.11.1 were released, the json i/o implementations were inappropriate. As a result, loading pandapower nets that were stored with newer versions may fail (->empty net).
The implementations of version v2.11.0 and v2.11.1 cannot be changed retrospectively.
Therefore, I propose to close this issue.
